### PR TITLE
Update header to 1.2.188

### PR DESCRIPTION
**Pull request by locke-lunarg**

### CODE REVIEW

**Review by patrick-lunarg — APPROVED** [review]

File: framework/decode/CMakeLists.txt [hunk 1]
```diff
@@ -130,12 +130,15 @@ target_sources(gfxrecon_decode
...
 )
 
-if (MSVC AND (MSVC_VERSION LESS 1920))
+if (MSVC)
     # These files may fail to compile with VS2017 and older, requiring the default section limit of 2^16 to be increased.
-    set_source_files_properties(${CMAKE_SOURCE_DIR}/framework/generated/generated_decode_pnext_struct.cpp PROPERTIES COMPILE_FLAGS /bigobj)
-    set_source_files_properties(${CMAKE_SOURCE_DIR}/framework/generated/generated_vulkan_replay_consumer.cpp PROPERTIES COMPILE_FLAGS /bigobj)
-    set_source_files_properties(${CMAKE_CURRENT_LIST_DIR}/vulkan_object_cleanup_util.cpp PROPERTIES COMPILE_FLAGS /bigobj)
-    set_source_files_properties(${CMAKE_CURRENT_LIST_DIR}/vulkan_replay_consumer_base.cpp PROPERTIES COMPILE_FLAGS /bigobj)
+    set_source_files_properties(${CMAKE_SOURCE_DIR}/framework/generated/generated_vulkan_struct_to_string.cpp PROPERTIES COMPILE_FLAGS /bigobj)
```
Comment (mikes-lunarg) [inline framework/decode/CMakeLists.txt:135]: I'm not sure why this `/bigobj` had to be split out from the others, it builds fine for me on 2019 without the option

File: framework/decode/vulkan_object_info_table_base.h [hunk 2]
```diff
@@ -0,0 +1,115 @@
+/*
+** Copyright (c) 2018-2020 Valve Corporation
+** Copyright (c) 2018-2020 LunarG, Inc.
```
Comment (mikes-lunarg) [inline framework/decode/vulkan_object_info_table_base.h:3]: 2021

File: framework/encode/vulkan_state_table_base.h [hunk 3]
```diff
@@ -0,0 +1,79 @@
+/*
+** Copyright (c) 2019 LunarG, Inc.
```
Comment (mikes-lunarg) [inline framework/encode/vulkan_state_table_base.h:2]: 2021

File: framework/generated/generated_vulkan_object_info_table_base2.h [hunk 4]
```diff
@@ -0,0 +1,295 @@
+/*
+** Copyright (c) 2018-2020 Valve Corporation
+** Copyright (c) 2018-2020 LunarG, Inc.
```
Comment (mikes-lunarg) [inline framework/generated/generated_vulkan_object_info_table_base2.h:3]: 2021

File: framework/generated/generated_vulkan_state_table.h [hunk 5]
```diff
@@ -0,0 +1,295 @@
+/*
+** Copyright (c) 2018-2020 Valve Corporation
+** Copyright (c) 2018-2020 LunarG, Inc.
```
Comment (mikes-lunarg) [inline framework/generated/generated_vulkan_state_table.h:3]: 2021

**Review by mikes-lunarg — APPROVED** [review]
Looks great! Just a few comments, mostly copyright dates.

File: framework/decode/CMakeLists.txt (hunk 1 above)
Comment (locke-lunarg) [reply framework/decode/CMakeLists.txt:135]: Because it failed in github ci, Windows MSVC Debug, if it didn't split.

**Pull request merged**